angular/projects/admin-nrpti/src/app/mines/mines-records-rows/mines-records-table-row.component.html

```diff
@@ -10,14 +10,12 @@
 <td scope="row" data-label="Collections" class="col-2 truncate">{{ rowData.collections && rowData.collections[0] ? rowData.collections[0].name : '-' }}</td>
 <td scope="row" data-label="Date" class="col-1 truncate">{{ getAttributeValue('dateIssued') }}</td>
 <td scope="row" data-label="Published" class="col-1 d-xl-flex">
-  <!-- TODO this publish flag needs to be updated based on BCMI publish (collections?) logic when ready.  Currently, -->
-  <!-- this will always show `unpublished` as the `public` role will never be found in the master read array -->
   <div *ngIf="isPublished(); then published; else unpublished"></div>
-  <ng-template #published>
+  <ng-template #published class="test">
```

Comment:
Curious about the class name of test, is that a unit test helper?

Reply:
Reasonable question. It's just a copy paste from the other places we use the pubished logic/css. Peeking into those CSS classes, there isn't actually a class called test that I can find, so likely a test helper, but it can also probably be removed without causing any issues. Tests pass without it.

Reply:
👍